packages/app/app/components/ArtistView/index.tsx

```diff
@@ -65,6 +71,22 @@ const ArtistView: React.FC<ArtistViewProps> = ({
                   { t('tour') }
                 </span>
               }
+
+              <a
+                href='#'
+                className={styles.artist_favorites_button_wrap}
+                data-testid='add-remove-favorite'
+                onClick={
+                  isFavorite
+                    ? removeFavoriteArtist
+                    : addFavoriteArtist
+                }
+              >
+                <Icon
+                  name={isFavorite ? 'star' : 'star outline'}
```

Comment:
Just noticed this recently that we have both stars and hearts for favorites. From now, let's just use hearts (and replace stars with hearts later).